.github/workflows/pr.yml

```
@@ -0,0 +1,33 @@
+name: Pull Request Description Validation
+
+on:
+  pull_request:
+    types: [opened, edited, reopened]
+
+jobs:
+  validate-pr-description:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check PR Description Against Template
+        id: prcheck
+        run: |
+          # Ensure jq is installed
+          sudo apt-get install jq
+
+          # Fetch PR description using jq
+          PR_DESCRIPTION=$(jq -r ".pull_request.body" "$GITHUB_EVENT_PATH")
+          KEYWORD="Bluth"
+
+          # Check for the required keyword in the PR description
+          if [[ $PR_DESCRIPTION = *"$KEYWORD"* ]]; then
+            echo "PR description does not have default text removed"
+            exit 1
+          fi
+      - name: Comment on PR
+        id: comment
+        if: failure() && steps.prcheck.outcome == 'failure'
+        uses: thollander/actions-comment-pull-request@v2
+        with:
+          message: |
+            Hi @${{ github.event.pull_request.user.login }}! Please remove the default text from the pull request description! 

```

The functionality is great, though maybe this message doesn't drive the behaviour we want. Maybe something like?
```suggestion
            Hi @${{ github.event.pull_request.user.login }}! It looks like you've left the default text in the pull request description. Please replace that text with something meaningful. Thanks!

```

---

yeah this is better. I'll accept it and test!